pkgs/development/interpreters/luajit/default.nix

```diff
@@ -22,27 +44,34 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     substituteInPlace Makefile --replace ldconfig :
+    if test -n "''${dontStrip-}"; then
+      # CCDEBUG must be non-empty or everything will be stripped, -g being
+      # passed by nixpkgs CC wrapper is insufficient on its own
+      substituteInPlace src/Makefile --replace "#CCDEBUG= -g" "CCDEBUG= -g"
+    fi
   '';
 
   configurePhase = false;
 
+  buildFlags = [
+    "amalg" # Build highly optimized version
```

Comment:
is there a downside to this ?

Reply:
To my knowledge, the only downside is that the *build* process with `amalg` enabled consumes more memory. `amalg` is the recommended target for binary distribution.